### ISSUE TITLE
Issue in multiple tabs

### ISSUE THREAD

**Issue by nijeeshvayalassery** [HackedByChinese/ng-idle#213]
If we use same application in multiple tabs, both application uses the same localstorage for storing ngIdle.expiry and one application extends the session of other one. Is there any work around to fix this issue in current version, like configure sessionstorage instead of localstorage or like change the id field in the ngIdle.expiry.  

Thanks in advance

**Comment by jkpulipati** [comment]
Yes, In our application too having the same issue. Can you please suggest how to achieve it. 
Thanks in advance

**Comment by menahemc** [comment]
Hey, did you resolve the issue?
I have the same problem.
thanks
